test: import apport-retrace to get coverage data

bdrung opened the pull request:
Calling `apport-retrace` as separate binary will prevent collecting code coverage data. So import the apport-retrace file and call the `main` function there.

laarmen reviewed (approved):
Arguably more importantly, it makes it much easier to debug when a test fails!

LGTM, assuming the tests end up passing? I don't have access to the test logs anymore.

bdrung committed:
test: import apport-retrace to get coverage data

Calling `apport-retrace` as separate binary will prevent collecting code
coverage data. So import the apport-retrace file and call the `main`
function there.

bdrung committed:
test: import apport-retrace to get coverage data

Calling `apport-retrace` as separate binary will prevent collecting code
coverage data. So import the apport-retrace file and call the `main`
function there.
